driver-mapping/src/main/java/com/datastax/driver/mapping/AnnotationParser.java

```diff
@@ -106,18 +112,24 @@ private AnnotationParser() {
         validateOrder(pks, "@PartitionKey");
         validateOrder(ccs, "@ClusteringColumn");
 
-        mapper.addColumns(createColumnMappers(pks, factory, mapper.entityClass, mappingManager, columnCounter),
-                createColumnMappers(ccs, factory, mapper.entityClass, mappingManager, columnCounter),
-                createColumnMappers(rgs, factory, mapper.entityClass, mappingManager, columnCounter));
+        mapper.addColumns(
+                createColumnMappers(pks, factory, mapper.entityClass, mappingManager, columnCounter, tableMetadata, ksName, tableName),
+                createColumnMappers(ccs, factory, mapper.entityClass, mappingManager, columnCounter, tableMetadata, ksName, tableName),
+                createColumnMappers(rgs, factory, mapper.entityClass, mappingManager, columnCounter, tableMetadata, ksName, tableName));
```

Comment:
That's true but frankly I prefer to leave it as is for practical reasons:

1. Quoting of case-sensitive identifiers is not very consistent, and is done in different places, so should we quote or not the result of `TableMetadata.getName()`?  The code coverage for case-sensitive use cases is not very good, so let's not introduce regressions for that.
2. In #686 I added specific tests for case-sensitiveness and also applied a methodic quoting of identifiers). See btw JAVA-564.
3. The less we modify this class the easier it will be to merge it with JAVA-541 :)